src/main/java/com/google/sps/data/Email.java

```diff
@@ -74,6 +77,34 @@ public static String addNewPost(Post post) throws IOException {
     return emailContent;
   }
 
+  /**
+    * Add all ranked posts into daily digest email.
+    *
+    * @param posts ranked from a college
+    * @return string of HTML file with 3 ranked posts
+    * @throws IOException
+    */
+  public static String addRankedPosts(ArrayList<Post> rankedPosts) throws IOException {
+
+    String emailContent = getStringFromHTML(dailyDigestPath);
+
+    // Split up email content to inject multiple posts in between.
+    int split = emailContent.indexOf("</p>") + 4;
```

Comment:
Make this a constant for readability. It's not immediately obvious why it should be 4.

Reply:
Gotcha!